enginetest/priv_auth_queries.go

```diff
@@ -287,6 +287,55 @@ var UserPrivTests = []UserPrivilegeTest{
 			},
 		},
 	},
+	{
+		Name: "Basic usage of flush privileges",
+		SetUpScript: []string{
+			"CREATE TABLE mydb.test (pk BIGINT PRIMARY KEY);",
+			"INSERT INTO mydb.test VALUES (1);",
+			"CREATE USER tester@localhost;",
+			"CREATE ROLE test_role;",
+			"GRANT SELECT ON mydb.* TO test_role;",
```

Comment:
I mentioned removing this test, but just as a note: make sure that each line in a test is there for a reason. The created role isn't mentioned anywhere else, and therefore creates a bit of noise when reading the test and having to see where it comes into play.

This also goes for the statement `SELECT * FROM mydb.test;/*1*/` further down. The comment `/*1*/` would make sense as there's another `SELECT * FROM mydb.test;` in the commented-out portion (assuming it's going to be uncommented someday), but `SELECT * FROM mydb.test2;` is only in one place in this test, therefore the comment is unnecessary. Looking at the test where this was initially done (which I'm assuming you modeled your test from)―`"Basic database and table name visibility"`―there's a comment by every statement that appears multiple times within the `Assertions` slice, and only on those statements.

Lastly, `mydb.test2` doesn't have any relevance to the test here, therefore the `SELECT` on it should have also been excluded.